README.md

```diff
@@ -16,11 +16,25 @@ In this modified Cell Painting, there are five channels:
 
 ## Plate maps
 
-We applied this modified Cell Painting assay using the following plate design:
+We applied this modified Cell Painting assay using the following plate design for the first two plates:
 
-![treatment_dose_platemap.png](4.analyze-data/platemap_figures/treatment_dose_platemap.png)
+- Plate 1 = **localhost220512140003_KK22-05-198**
 
-See our [platemaps](3.process-cfret-features/metadata/) for more details.
+![plate3_CFReT.png](example_figs/plate1_CFReT.png)
+
+- Plate 2 = **localhost220513100001_KK22-05-198_FactinAdjusted**
+
+![treatment_dose_platemap.png](4.analyze_data/platemap_figures/treatment_dose_platemap.png)
+
+For the third plate, we are using the following plate design:
+
+- Plate 3 = **localhost230405150001**
```

Comment:
Is this the name of the plate? I am but confused about what localhost is.

Reply:
Ahh yes, so this will be important once we can get the data public on Figshare. This is the name of the plate from the folder name when the data was uploaded.

The actual naming convention of `localhost` I don't understand either but that is the name of the folder that the images come from. I associate that name to a short name with is Plate #.